src/com/palmergames/bukkit/config/ConfigNodes.java

```diff
@@ -618,6 +621,9 @@
             "global_nation_settings.default.open",
             "false",
             "# If set to true, any newly made nation will have open status and any town may join without an invite."),
+	GNATION_DEF_BOARD("global_nation_settings.default.board", 
+			"/nation set board [msg]",
+			"# Default nation board"),

```

Comment:
This should be moved to a new Config section "nation" akin to the default Town settings section "town". The new section should be below the existing "town" section. Let me know if that doesn't make sense.

The 1st line highlighted above would then be `NATION_DEF_BOARD("nation.default_board",`

Reply:
Done, tested my newest commit and it seems to work fine.